src/main/java/org/blocovermelho/ae2emicrafting/client/Ae2EmiPlugin.java

```diff
@@ -16,32 +14,33 @@
 import dev.emi.emi.api.recipe.VanillaEmiRecipeCategories;
 import dev.emi.emi.api.stack.EmiIngredient;
 import dev.emi.emi.api.stack.EmiStack;
-import dev.emi.emi.api.widget.Bounds;
-import org.blocovermelho.ae2emicrafting.client.handler.Ae2PatternTerminalDragHandler;
-import org.blocovermelho.ae2emicrafting.client.handler.Ae2PatternTerminalHandler;
-import org.blocovermelho.ae2emicrafting.client.handler.Ae2RecipeHandler;
+import org.blocovermelho.ae2emicrafting.client.handler.*;
+import org.blocovermelho.ae2emicrafting.client.handler.generic.Ae2BaseScreenExclusionZones;
+import org.blocovermelho.ae2emicrafting.client.handler.generic.Ae2BaseStackProvider;
+import org.blocovermelho.ae2emicrafting.client.handler.generic.Ae2PatternTerminalDragHandler;
+import org.blocovermelho.ae2emicrafting.client.helper.mapper.EmiFluidStackConverter;
+import org.blocovermelho.ae2emicrafting.client.helper.mapper.EmiItemStackConverter;
+import org.blocovermelho.ae2emicrafting.client.helper.mapper.EmiStackConverters;
 import org.blocovermelho.ae2emicrafting.client.recipes.*;
 
 import java.util.List;
 
 public class Ae2EmiPlugin implements EmiPlugin {
     @Override
     public void register(EmiRegistry registry) {
+        EmiStackConverters.register(new EmiItemStackConverter());
+        EmiStackConverters.register(new EmiFluidStackConverter());
+
+        registry.addGenericExclusionArea(new Ae2BaseScreenExclusionZones());
+        registry.addGenericStackProvider(new Ae2BaseStackProvider());
```

Comment:
There is a regression that was caused by this line.
I see that you made this for like dragging things inside the grid but it is conflicting with AE2 when it comes to fluids.

Is there any other reasoning why `AE2BaseStackProvider` was added, like this is what happens with the official JEI/REI integration?
```suggestion
//      registry.addGenericStackProvider(new Ae2BaseStackProvider());
```